OpenRA.Utility/Program.cs

```diff
@@ -78,13 +89,13 @@ static void Main(string[] args)
 				Log.Write("utility", "Received args: {0}", args.JoinWith(" "));
 				Log.Write("utility", "{0}", e);
 
-			    if (e is ArgumentException)
-			        Console.WriteLine("No such command '{0}'", args[0]);
-			    else
-			    {
-                    Console.WriteLine("Error: Utility application crashed. See utility.log for details");
-                    throw;
-			    }
+				if (e is NoSuchCommandException)
+					Console.WriteLine(e.Message);
```

Comment:
This cleanup enables you to `catch (NoSuchCommandException)` and do the error handling there, which would be better.